packages/api/package.json

```diff
@@ -51,13 +51,17 @@
     "get-stream": "^6.0.1",
     "isomorphic-fetch": "^3.0.0",
     "js-yaml": "^4.1.0",
-    "json-schema-to-typescript": "^10.1.5",
+    "json-schema-to-ts": "^2.5.5",
     "json-schema-traverse": "^1.0.0",
+    "lodash.camelcase": "^4.3.0",
+    "lodash.deburr": "^4.1.0",
     "lodash.merge": "^4.6.2",
+    "lodash.setwith": "^4.3.2",
+    "lodash.startcase": "^4.4.0",
     "make-dir": "^3.1.0",
-    "oas": "^19.0.0",
-    "object-hash": "^3.0.0",
+    "oas": "^19.0.1",
     "ora": "^5.4.1",
+    "prettier": "^2.7.1",
```

Comment:
`json-schema-to-typescript` had Prettier as a dependency for formatting but since we aren't using that library anymore we need to now load in Prettier ourselves here so we can format the code we're generating. `ts-morph` by default doesn't generate code that looks great.